app/server/utils/user.py

```diff
@@ -202,8 +202,7 @@ def create_transfer_account_user(first_name=None, last_name=None, preferred_lang
         user.set_held_role('BENEFICIARY', 'beneficiary')
 
     if organisation:
-        user.organisations.append(organisation)
```

Comment:
@enjeyw A user (non-admin) is tied to an org but they won't have the transfer account associated to them?